openwisp_radius/admin.py

```diff
@@ -386,6 +386,7 @@ class RadiusPostAuthAdmin(OrganizationFirstMixin, BasePostAuth):
 @admin.register(RadiusBatch)
 class RadiusBatchAdmin(MultitenantAdminMixin, TimeStampedEditableAdmin):
     change_form_template = 'openwisp-radius/admin/rad_batch_users_change_form.html'
+    add_form_template = 'openwisp-radius/admin/rad_batch_users_add_form.html'
```

Comment:
looks like this CSS is missing: https://github.com/openwisp/openwisp-utils/blob/master/openwisp_utils/admin.py#L163.

See how it looks: https://github.com/openwisp/openwisp-utils#openwisp-utils-admin-helptextstackedinline.